adagrams/game.py

```diff
@@ -1,13 +1,126 @@
 from random import randint
 
 def draw_letters():
-    pass
+    tiles = [
+    'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 
+    'B', 'B',
+    'C', 'C',
+    'D', 'D', 'D', 'D', 
+    'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 
+    'F', 'F', 
+    'G', 'G', 'G', 
+    'H', 'H',  
+    'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 
+    'J',  
+    'K', 
+    'L', 'L', 'L', 'L', 
+    'M', 'M', 
+    'N', 'N', 'N', 'N', 'N', 'N', 
+    'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 
+    'P', 'P', 
+    'Q', 
+    'R', 'R', 'R', 'R', 'R', 'R', 
+    'S', 'S', 'S', 'S', 
+    'T', 'T', 'T', 'T', 'T', 'T',
+    'U', 'U', 'U', 'U', 
+    'V', 'V', 
+    'W', 'W', 
+    'X', 
+    'Y', 'Y', 
+    'Z'
+    ]
+    selected_index = []
+    picked_letters = []
+    for i in range(0, 10):
+        rand_selection = randint(0, len(tiles)-1)
+        while rand_selection in selected_index:
+            rand_selection = randint(0, len(tiles)-1)
+        selected_index.append(rand_selection)
+        random_letter = tiles[rand_selection]
+        picked_letters.append(random_letter)
+    return picked_letters
+
 
 def uses_available_letters(word, letter_bank):
-    pass
+    word_uppercase = word.upper()
+    letter_frequency_word = {}
+    letter_frequency_hand = {}
+    
+    for i in word_uppercase:
+        if i not in letter_bank:
+            return False
+        else:
+            letter_frequency_word[i] = letter_frequency_word.get(i, 0) + 1
```

Comment:
We don't need an else condition here and we can remove it to simplify the conditional logic. 

If a letter is not in the `letter_bank` then you return `False` and the function stops its execution. If the condition on line 50 is not true and you don't return False, then we want the code on line 53 to execute. 

This allows you to remove else, which matters because very line of code is another place where a bug can be created and it reduces the conditional complexity of your code.

```suggestion
        if i not in letter_bank:
            return False
        letter_frequency_word[i] = letter_frequency_word.get(i, 0) + 1
```